qt/applications/workbench/workbench/widgets/settings/plots/presenter.py

```diff
@@ -230,6 +239,32 @@ def action_default_colormap_changed(self):
 
         ConfigService.setString(PlotProperties.COLORMAP.value, colormap)
 
+    def populate_font_combo_box(self):
+        font_list = font_manager.findSystemFonts()
+        fonts = set()
+        for font in font_list:
+            # This try-except is for a known matplotlib bug where get_name() causes an error for certain fonts.
+            try:
+                font_name = font_manager.FontProperties(fname=font).get_name()
+            except RuntimeError:
+                continue
+            fonts.add(font_name)
+
+        if mpl.rcParams['font.family'][0] in ['sans-serif', 'serif', 'cursive', 'fantasy', 'monospace']:
+            current_mpl_font = mpl.rcParams['font.' + mpl.rcParams['font.family'][0]][0]
+        else:
+            current_mpl_font = mpl.rcParams['font.family'][0]
+        fonts.add(current_mpl_font)
+        fonts = sorted(fonts)
+
+        self.view.plot_font.addItems(fonts)
```

Comment:
I'm a little concerned this presenter is starting to grow into a model-presenter combo:
Everything from 242-259 should be in a model, and then the presenter can focus on just the view